docker/docker-compose-sbt.yml

```diff
@@ -0,0 +1,10 @@
+# Base container for sbt services. In a seperate file so that a container doesn't get created for it by default.
+sbt:
+  build: .
+  dockerfile: Dockerfile-sbt
+  restart: unless-stopped
+  volumes:
+    - "./entrypoints:/entrypoints"
+  entrypoint: /entrypoints/sbt.sh
+  expose:
+    - "9000"
```

Comment:
all sbt projects are run on port 9000 in their container with the nginx container operating as a proxy.